src/utils/utils.py

```diff
@@ -25,18 +23,17 @@
 from allennlp.modules.seq2seq_encoders.seq2seq_encoder import Seq2SeqEncoder
 from allennlp.common.params import Params
 
+from .tokenizers import OpenAIBPETokenizer, MosesTokenizer
```

Comment:
Maybe we should move the data loaders out of `utils.py` and into another module, so that `utils.py` can stay a low-level thing that doesn't have any other dependencies witihin jiant?